qase-testcafe/src/index.ts

```diff
@@ -217,13 +221,13 @@ class TestcafeQaseReporter {
         }
     };
 
-    public reportTaskDone = async () => {
+    public reportTaskDone = async (endTime, passed, warnings, result) => {
```

Comment:
please take a look here, linter says you define vars but never use them

Reply:
It is necessary to declare function with parameters, despite of there are unused, because when function signature not like expected, testcafe does not call this function

Reply:
@kmalinin  Please add the eslint-ignore rule for this line and comment about the function signature.